packages/design-system/src/lib/components/card/Card.tsx

```diff
@@ -1,17 +1,40 @@
-import { PropsWithChildren } from 'react'
 import { Card as CardBS } from 'react-bootstrap'
 import { CardHeader } from './CardHeader'
 import { CardBody } from './CardBody'
+import { CardImage } from './CardImage'
 
 /**
  * ## Description
  * The card component shows a body in a frame with a header.
  */
-function Card({ children }: PropsWithChildren) {
-  return <CardBS>{children}</CardBS>
+
+interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode
+  border?: 'primary' | 'secondary' | 'success' | 'danger' | 'warning' | 'info' | 'dark' | 'light'
```

Comment:
should the design token colors be updated to reflect these choices for border and text?  For example, we currently don't have colors for 'dark' and 'light'